target/linux/kirkwood/base-files/lib/upgrade/platform.sh

```diff
@@ -16,6 +16,13 @@ platform_do_upgrade() {
 	linksys,ea4500)
 		platform_do_upgrade_linksys "$1"
 		;;
+	netgear,readynas-duo-v2)
+		mkdir -p /var/lock
+		touch /var/lock/fw_printenv.lock
+		fw_setenv bootargs 'console=ttyS0,115200n8 earlyprintk'
+		fw_setenv bootcmd 'nand read.e 0x1200000 0x200000 0x600000;bootm 0x1200000'
```

Comment:
Hmm... Can we run these commands at installation time and drop these lines here?

Reply:
Ok, I will try detect if sysupgrade is called from initramfs and run it only then. But I could test it after 2-3 months, because my device is away from me now.

Reply:
I must think about it in the next week.

Reply:
@adschm @mans0n 
Any ideas, how to detect if sysupgrade is running from initramfs image?

Reply:
I'm not sure... But I thought we can just run those commands in U-Boot shell before the tftpboot, am I missing something?

Reply:
Or just check if they match already. Either would be fine by me.

Reply:
Alternatively, maybe at least the "bootargs" part could be added to device tree as "chosen" property in /aliases in device tree?

Reply:
I prepared new version: envs are configured only during first installation via initramfs. If someone want override it, changes wouldn't be destroyed via regular sysupgrade.